packages/gatsby-theme-carbon/src/components/MiniCard/CardGroup.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Row, Column } from '../Grid';
+import { cardGroup } from './CardGroup.module.scss';
+
+const CardGroup = ({ children, className, ...rest }) => (
```

Comment:
Hmm I thought we would just give them a cardGroup class? Can't we just do 
```css
.cardGroup .column
```
So we don't have to map over the children? Following the pattern of DoDontRow